content/en/core/releases/3.14.0.md

```diff
@@ -0,0 +1,143 @@
+# 3.14.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [UI/UX changes](#uiux-changes)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+#### New dialog showing sync status
+
+When an offline user manually triggers a sync via the hamburger menu, a notification dialog is opened indicating the status of the sync process.
+
+Sync in progress:
+![](../images/3.14.0-5207-current-sync.png)
+
+Sync complete:
+![](../images/3.14.0-5207-sync-done.png)
+
+Sync failed (with retry option):
+![](../images/3.14.0-5207-sync-failed.png)
+
+[#5207](https://github.com/medic/cht-core/issues/5207): Show dialog when user clicks "Sync Now" in hamburger menu
+
+#### Updated icon for the Create Report button
+
+The icon for the Create Report button has been updated so that its purpose is more clear.
+
+Old Icon | New Icon
+:-------------------------:|:-------------------------:
+![](../images/3.14.0-6403-old.png)  |  ![](../images/3.14.0-6403-new.png)
```

Comment:
I can't add a commend to the image so, I'll add it here. The "3.14.0-6403-old.png" has the word "Areas" in the background which is a bit confusion - I think it's just showing through the partial transparency and nothing to do with the actual change. If it's not too much effort can you replace it with a screenshot with a plain background?